sevenhills.html

```diff
@@ -0,0 +1,290 @@
+---
+layout: default
+---
+
+<section class="jumbotron jumbotron-basics hard-top flush-bottom" data-optimize-bg-img data-hero-image>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12">
+                <h1 class="component-header-box component-header-box-border-lg mt-100">
```

Comment:
`mt-100` looks like a BS4 convention. I don't see any references to that selector in this PR, `crds-net` or `crds-styles`. Can you please remove this unless you've defined that selector somewhere else?

Reply:
Yea, this is more of a BS4 thing I used across a lot of these pages. I added it in the _utilities.scss on the our-history branch